packages/taquito-remote-signer/test/taquito-remote-signer.spec.ts

```diff
@@ -125,7 +125,96 @@ describe('RemoteSigner test', () => {
         signer.sign(
           '0365cac93523b8c10346c0107cfea5e12ff3c759459020e532f299e2f41082f7cb6d0000f68c4abfa21dfc0c9efcf588190388cac85d9db60f81d6038b79d8030000000000b902000000b405000764045b0000000a2564656372656d656e74045b0000000a25696e6372656d656e740501035b0502020000008503210317057000010321057100020316072e020000002b032105700002032105710003034203210317057000010321057100020316034b051f020000000405200002020000002b0321057000020321057100030342032103170570000103210571000203160312051f0200000004052000020321053d036d0342051f020000000405200003000000020000'
         )
-      ).rejects.toThrowError(/Signature failed verification against public key/);
+      ).rejects.toThrowError(
+        /Signature failed verification against public key/
+      );
+
+      done();
+    });
+
+    it('Should fail if pkh is invalid', async (done) => {
+      try {
+        const signer = new RemoteSigner(
+          'tz9iD5nmudc4QtfNW14WWaiP7JEDuUHnbXuv',
+          'http://127.0.0.1:6732',
+          {},
+          httpBackend as any
+        );
+        await expect(
```

Comment:
Lines 143-149 are irrelevant for the test, the error is thrown at instantiation.

Reply:
removed unneeded lines of code from the test